Tasks/AzureVmssDeployment/task.json

```diff
@@ -33,6 +29,17 @@
             "displayName": "Image Details",
             "isExpanded": true,
             "visibleRule": "action = Update image || action = UpdateImage"
+        },
+        {
+            "name": "StartupConfiguration",
+            "displayName": "Configure start-up",
+            "isExpanded": true,
+            "visibleRule": "action = Configure application startup || action = Update image || action = UpdateImage"
```

Comment:
this is to support back-compat. Earlier action was "UpdateImage" which was renamed to "Update image" a few sprints back. So, some of the users tasks might still be using "UpdateImage".